app/models/public_body.rb

```diff
@@ -68,6 +68,11 @@ class PublicBody < ActiveRecord::Base
     }
 
     translates :name, :short_name, :request_email, :url_name, :notes, :first_letter, :publication_scheme
+    accepts_nested_attributes_for :translations, :reject_if => :empty_translation_in_params?
```

Comment:
This looks like it should get handled by [`Translatable`](https://github.com/mysociety/alaveteli/blob/develop/app/models/concerns/translatable.rb#L4-L6).